Dockerfile

```diff
@@ -33,6 +33,8 @@ RUN \
   npm install pm2@3.2.2 -g && \
   echo "INFO: npm install --production..." && \
   cd /opt/sth && npm install --production && \
+  mkdir temp && \
```

Comment:
Maybe it would be useful to add over this line:

```
# Directory used by STH to generate csv files
```

otherwise it is easy to forget in the future why this directory is necessary (the name "temp" doesn't provide any clue)